### DISCUSSION TITLE
在UE5.3上使用会出现Failed to deserialize the V8 snapshot blob.

### DISCUSSION THREAD

**Discussion by watsonsong** [Tencent/puerts#1570] [Q&A] [answered]
直接使用[Unreal_v1.0.5](https://github.com/Tencent/puerts/releases/tag/Unreal_v1.0.5)的版本在UE5.3上编译不过，使用了master的代码配合这个发布包的v8库。然后打开AutoModeEnable之后，在创建JsEnv的时候会产生如下报错：
`Failed to deserialize the V8 snapshot blob. This can mean that the snapshot blob file is corrupted or missing.`

后来也测试了[V8_9.4.146.24_230822](https://github.com/puerts/backend-v8/releases/tag/V8_9.4.146.24_230822)，看Commit时间这个已经去掉SnapshotBlob.h的拷贝。但使用之后依然会有这个报错。

请教一下是否[这个Commit](https://github.com/Tencent/puerts/commit/9f66dc8c2c543126fe7c684ac8030ab24d85fb59)没有编译的里面。是因为需要重新编译v8的库的问题么？

上传了一个测试的工程：https://github.com/watsonsong/TestPuerts

**Comment by chexiongsheng** [comment, marked as answer]
使用最新的puerts代码加上最新v8是没问题的，我一直在用。
你这个估计是用了老的puerts代码（会设置snapshot）配了新的v8导致。
建议”完整“删除ThirdParty下v8目录

**Comment by watsonsong** [comment]
多谢，我全部清理重新编译之后好了，估计是旧的obj之类还在被链接导致的。
